src/node/Main.hs

```diff
@@ -109,77 +121,139 @@ action
+                    wDhtWorkers = first (map $ wrapActionSpec $ "worker" <> "dht")
+                                        (dhtWorkers (rmGetPeers liftedTransport) kademliaInst)
+                let allPlugins :: ([WorkerSpec (StatsMode SscGodTossing)], OutSpecs)
+                    allPlugins = mconcat [ wDhtWorkers
+                                         , convPlugins currentPluginsGT
+                                         , utwStats]
+                runNodeStats @SscGodTossing
+                    (CLI.peerId commonArgs)
+                    liftedTransport
+                    allPlugins
+                    currentParams gtParams
+            (True, NistBeaconAlgo) -> do
+                let liftedTransport = hoistResources lift transport
+                    wDhtWorkers = first (map $ wrapActionSpec $ "worker" <> "dht")
+                                        (dhtWorkers (rmGetPeers liftedTransport) kademliaInst)
+                let allPlugins :: ([WorkerSpec (StatsMode SscNistBeacon)], OutSpecs)
+                    allPlugins = mconcat [ wDhtWorkers
+                                         , convPlugins currentPlugins
+                                         , utwStats]
+                runNodeStats @SscNistBeacon
+                    (CLI.peerId commonArgs)
+                    (hoistResources lift transport)
+                    allPlugins
+                    currentParams ()
+            (False, GodTossingAlgo) -> do
+                let liftedTransport = hoistResources lift transport
+                    wDhtWorkers = first (map $ wrapActionSpec $ "worker" <> "dht")
+                                        (dhtWorkers (rmGetPeers liftedTransport) kademliaInst)
+                let allPlugins :: ([WorkerSpec (ProductionMode SscGodTossing)], OutSpecs)
+                    allPlugins = mconcat [ wDhtWorkers
+                                         , convPlugins currentPluginsGT
+                                         , utwProd]
+                runNodeProduction @SscGodTossing
+                    (CLI.peerId commonArgs)
+                    liftedTransport
+                    allPlugins
+                    currentParams gtParams
+            (False, NistBeaconAlgo) -> do
+                let liftedTransport = hoistResources lift transport
+                    wDhtWorkers = first (map $ wrapActionSpec $ "worker" <> "dht")
+                                        (dhtWorkers (rmGetPeers liftedTransport) kademliaInst)
+                let allPlugins :: ([WorkerSpec (ProductionMode SscNistBeacon)], OutSpecs)
+                    allPlugins = mconcat [ wDhtWorkers
+                                         , convPlugins currentPlugins
+                                         , utwProd]
+                runNodeProduction @SscNistBeacon
+                    (CLI.peerId commonArgs)
+                    liftedTransport
+                    allPlugins
+                    currentParams ()
```

Comment:
But to be honest, I don't know how to get rid of code duplication.
I expect it would be less when we merge master into this branch.

this code duplication is caused by we used same transformers stack (as for full node) for wallet-node before, but now it's important to use wallet's transformers stack (with WalletWebDb)